helmfile.d/charts/prometheus-alerts/templates/alerts/kubernetes-system-kubelet.yaml

```
@@ -176,28 +176,85 @@ spec:
 {{- if .Values.defaultRules.additionalRuleLabels }}
 {{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
 {{- end }}
-    - alert: LessKubeletsThanNodes
+{{- if .Values.autoscaledNodeGroupAlerts.enabled }}
+    - alert: KubletDownForAutoscaledNodeFor15m
       annotations:
-        description: There are less kubelets than nodes for the last 5m.
-        runbook_url: {{ .Values.defaultRules.runbookUrl }}kubernetes/LessKubeletsThanNodes
-        summary: There are less kubelets than nodes for the last 5m.
-      expr: up{job="kubelet", metrics_path="/metrics"} == 0
+        description: The kubelet job of an existing auntoscaled node {{`{{`}} $labels.node {{`}}`}} in cluster {{`{{`}} $labels.cluster {{`}}`}} is down for the last 15m.
```

Can't we do this? Also noticed a typo
```suggestion
        description: The kubelet job of an existing autoscaled node {{`{{ $labels.node }}`}} in cluster {{`{{ $labels.cluster }}`}} is down for the last 15m.
```

---

Typo fixed. I checked and both expression work. Changed to a simpler one.